en/appendices/4-4-migration-guide.rst

```diff
@@ -96,6 +96,18 @@ be removed in 5.0.
 - Use :ref:`controller-viewclasses` instead of defining view class mappings in
   ``RequestHandlerComponent``.
 
+The automatic view switching for 'ajax' requests offered by
```

Comment:
Isn't it deprecated instead of not available?

Reply:
Was this supposed to go into 5.0?

Reply:
Thanks! got that fixed.